lib/pbench/test/unit/server/query_apis/commons.py

```diff
@@ -266,9 +292,14 @@ def missing_key_helper(keys):
                 == f"Missing required parameters: {','.join(missing)}"
             )
 
-        parameter_items = self.cls_obj.schemas[
-            API_METHOD.POST
-        ].body_schema.parameters.items()
+        if self.api_method == API_METHOD.GET:
+            parameter_items = self.cls_obj.schemas[
+                self.api_method
+            ].uri_schema.parameters.items()
+        elif self.api_method == API_METHOD.POST:
+            parameter_items = self.cls_obj.schemas[
+                self.api_method
+            ].body_schema.parameters.items()
```

Comment:
I think Dave mentioned this elsewhere, but here's the code for it:  this is assuming that `POST` requests have parameters only in the body and none in the URI nor query parameters.

Also, should we have more branches to this `if` statement, e.g., for `PUT` and/or `DELETE`?  (If not, should we have an "else fail"??)